data/json/items/tools.json

```diff
@@ -3414,7 +3414,15 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "weapons",
-    "use_action": "MOLOTOV"
+    "use_action": {
+        "type": "transform",
+        "target": "MOLOTOV_LIT",
```

Comment:
You're right. I had changed it back, just not in my actual workspace.... Thanks for pointing that out.